databases/week1/db1-homework-my_squirrel.sql

```diff
@@ -0,0 +1,85 @@
+-- Hello My_SQuirreLs, here is my homework ---> databases-1 (a.k.a. week 15) ---> Magda /HYF#12
+
+USE hyf_db_1;   -- all the comments/questions will be removed once resolved :) 
+SELECT * FROM status; 
+SELECT * FROM task;  --  jest powiązany z tabelą status przez status_id i z tabelą user przez user_id
+SELECT * FROM user;   
+
+-- 01 -- Find out how many tasks are in the task table: 
+SELECT count(id) AS `number_of_tasks_in_task_table`   -- backticks work with column aliases but not the values!!!
```

Comment:
it also works without the backticks ;)

Reply:
Noted! Thank you. 
BTW forgive me the notes in Polish :D

Reply:
i'm not sure i can forgive that xD